apps/backend/src/app/entities/Purchase.ts

```diff
@@ -34,6 +36,8 @@ export class Purchase {
   @Column('int', { name: 'num_items' })
   numItems: number;
 
+  // TODO MAKE THIS WORK
+  @Transform(purchasedAt => moment(purchasedAt).format('MM/DD/YY'))
```

Comment:
trying to figure out how to make the date format look a bit better. Currently strings like `2021-02-28T05:00:00.000Z`